share_logs.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+for log in "/tmp/enigma_libegm.log" "/tmp/enigma_compiler.log" "/tmp/enigma_compile.log";
+do
+  echo -n "$log: "
+  cat $log | curl -F 'sprunge=<-' http://sprunge.us
```

Comment:
Like ten things are calling this. Shouldn't you delete the files to prevent their reupload?